lib/rubocop/cop/lint/ruby2_keywords.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop looks for `ruby2_keywords` calls for methods that do not need it.
+      #
+      # `ruby2_keywords` should only be called on methods that accept an argument splat
+      # (`*args`) but do not explicit keyword arguments (`k:` or `k: true`) or
+      # a keyword splat (`**kwargs`).
+      #
+      # @example `ruby2_keywords` given a `def`
+      #   # bad
+      #   ruby2_keywords def foo
+      #   end
+      #
+      #   ruby2_keywords def foo(arg)
+      #   end
+      #
+      #   ruby2_keywords def foo(**args)
+      #   end
+      #
+      #   ruby2_keywords def foo(i:, j:)
+      #   end
+      #
+      #   ruby2_keywords def foo(*args, i:, j:)
+      #   end
+      #
+      #   ruby2_keywords def foo(*args, **kwargs)
+      #   end
+      #
+      #   # good
+      #   ruby2_keywords def foo(*args)
+      #   end
+      #
+      # @example `ruby2_keywords` given a method name
+      #   # The same argument formats as above apply
+      #
+      #   # bad
+      #   def foo
+      #   end
+      #   ruby2_keywords :foo
+      #
+      #   # good
+      #   def foo(*args)
+      #   end
+      #   ruby2_keywords :foo
```

Comment:
A good case compatible with a bad case would be to remove `ruby2_keywords`.

```suggestion
      #   def foo
      #   end
```

Reply:
And I think it will be easier to understand whether this case is represented at the top of example.